pyeantic fails to parse rational field

Currently, the following does not work:

```py
sage: from pyeantic import RealEmbeddedNumberField
sage: RealEmbeddedNumberField(QQ).renf
NumberField(x-1, 1.000000000000000000000000000000000000000000000000000000000000000)
sage: K = RealEmbeddedNumberField(QQ).renf
sage: RealEmbeddedNumberField(K)
---------------------------------------------------------------------------
AssertionError                            Traceback (most recent call last)
<ipython-input-5-d72244347b0b> in <module>
----> 1 RealEmbeddedNumberField(K)

/tmp/ruth/micromamba/envs/flatsurvey/lib/python3.9/site-packages/pyeantic/__init__.py in RealEmbeddedNumberField(*args, **kwargs)
     60     """
     61     from .real_embedded_number_field import RealEmbeddedNumberField as RENF
---> 62     return RENF(*args, **kwargs)

/tmp/ruth/micromamba/envs/flatsurvey/lib/python3.9/site-packages/sage/misc/classcall_metaclass.pyx in sage.misc.classcall_metaclass.ClasscallMetaclass.__call__ (build/cythonized/sage/misc/classcall_metaclass.c:1743)()
    320         """
    321         if cls.classcall is not None:
--> 322             return cls.classcall(cls, *args, **kwds)
    323         else:
    324             # Fast version of type.__call__(cls, *args, **kwds)

/tmp/ruth/micromamba/envs/flatsurvey/lib/python3.9/site-packages/pyeantic/real_embedded_number_field.py in __classcall__(cls, embed, category)
    376             import re
    377             match = re.match("^NumberField\\(([^,]+), (\\[[^\\]]+\\])\\)$", repr(embed))
--> 378             assert match, "renf_class printed in an unexpected way"
    379             minpoly = match.group(1)
    380             root_str = match.group(2)

AssertionError: renf_class printed in an unexpected way
```

--- a/doc/news/parse.rst
+++ b/doc/news/parse.rst
@@ -1,0 +1,3 @@
+**Fixed:**
+
+* Fixed parsing of trivial number fields in pyeantic. #197

--- a/pyeantic/src/pyeantic/real_embedded_number_field.py
+++ b/pyeantic/src/pyeantic/real_embedded_number_field.py
@@ -382,6 +382,15 @@ class RealEmbeddedNumberField(UniqueRepresentation, CommutativeRing):
             sage: K = eantic.renf_class.make("x^2 - 2", "x", "1.4 +/- 1")
             sage: L = RealEmbeddedNumberField(K)
 
+        TESTS:
+
+        Check that #197 has been resolved::
+
+            sage: from pyeantic import RealEmbeddedNumberField
+            sage: K = RealEmbeddedNumberField(QQ).renf
+            sage: RealEmbeddedNumberField(K)
+            Real Embedded Number Field in x with defining polynomial x - 1 with x = 1
+
         """
         if 'cppyy.gbl.boost.intrusive_ptr<const eantic::renf_class>' in str(type(embed)):
             embed = embed.get()
@@ -390,12 +399,12 @@ class RealEmbeddedNumberField(UniqueRepresentation, CommutativeRing):
             # the printed representation of the renf_class. This is of course
             # not very robust…
             import re
-            match = re.match("^NumberField\\(([^,]+), (\\[[^\\]]+\\])\\)$", repr(embed))
-            assert match, "renf_class printed in an unexpected way"
+            match = re.match("^NumberField\\(([^,]+), ([^)]+)\\)$", repr(embed))
+            assert match, "renf_class printed in an unexpected way: " + repr(embed)
             minpoly = match.group(1)
             root_str = match.group(2)
-            match = re.match("^\\d*\\*?([^\\^ *]+)[\\^ ]", minpoly)
-            assert match, "renf_class printed leading coefficient in an unexpected way"
+            match = re.match("^\\d*\\*?([^\\^ *]+)[\\^ +-]", minpoly)
+            assert match, "renf_class printed leading coefficient in an unexpected way: " + minpoly
             minpoly = QQ[match.group(1)](minpoly)
             roots = []
             AA_roots = minpoly.roots(AA, multiplicities=False)

